src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -0,0 +1,74 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
```

Comment:
I think this needs `StatusQ.Components 0.1` import as well

Reply:
> Sure I will add a page for it. Its being used at so many places though and thought it would be okay not too

Actually that's fair enough

> Its works without it or
what do you mean?

Surprisingly it seems to work